manifests/init.pp

```diff
@@ -36,6 +36,7 @@
   # interval in hours, purging enabled when >= 1
   $purge_interval   = 0,
   # log4j properties
+  $log4j_file   = undef,
```

Comment:
I will add docstring, and yeah, deployment (of target) is outside the scope, so just the link creation is in this module.